e2eTests/newUser.e2e.js

```diff
@@ -5,7 +5,7 @@ fixture`Electron test`
 
 const longTimeout = 100000
 
-test('User can create new community and register', async t => {
+test('User can create new community, register and send few messages to general channel', async t => {
```

Comment:
Currently it's one big use case. I could use the fact that I have trouble creating separate use cases (with testcafe and electron) and maybe move sending message to other `test` scope so it would be more readable. I have to make sure that those tests are being run in correct order though.